flytekit/deck/deck.py

```diff
@@ -174,3 +174,29 @@ def get_deck_template() -> "Template":
         autoescape=select_autoescape(enabled_extensions=("html",)),
     )
     return env.get_template("template.html")
+
+
+class PythonDependencyDeck(Deck):
+    """
+    PythonDependencyDeck is a deck that contains information about the library name and version.
+    """
+
+    def __init__(self, name: Optional[str] = "Python Dependency", html: Optional[str] = ""):
+        super().__init__(name, html)
+
+    @property
+    def html(self) -> str:
+        import json
+        import subprocess
+
+        import pandas as pd
+        from flytekitplugins.deck.renderer import TableRenderer
+
+        try:
+            installed_packages = json.loads(subprocess.check_output(["pip", "list", "--format", "json"]))
```

Comment:
This is the [recommended way](https://pip.pypa.io/en/stable/user_guide/#using-pip-from-your-program) of calling into `pip`: 

```suggestion
            installed_packages = json.loads(subprocess.check_output([sys.executable, "-m", "pip", "list", "--format", "json"]))
```

Reply:
updated